src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/PanelMemberSpecialismsMapping.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
```

Comment:
I don't think this should have been pulled in

Reply:
I will close this PR